CTP/README.md

```diff
@@ -1,25 +1,23 @@
 # CTP - CUBRID Test Program
 
 ## Introduction
-CTP is a testing tool for an open source project CUBRID. It is written in Java and easy to execute tests with a simple configuration. 
+CTP is a testing tool for an open source project CUBRID. It is written in Java and easy to execute tests with a simple configuration 
 
 
 ## Requirements
-* It supports Linux and Windows (Cygwin is required).
-* Install Java 6 or higher version, and you also need to set ``JAVA_HOME`` environment variable to point to the installation directory.
-* CUBRID and CUBRID_DATABASES environment variables should be configured before executing testing, please refer to http://www.cubrid.org/ for configurations.
-* CUBRID QA executes testing for SQL and Medium on linux is based on ha mode, so you must make sure ports in cubrid.conf, cubrid_broker.conf and cubrid_ha.conf will
-  not conflict with another instance exists. Otherwise, start server or broker will be fail.
+* It supports Linux and Windows (Cygwin is required)
+* Install Java 6 or higher version, and you also need to set ``JAVA_HOME`` environment variable to point to the installation directory
+* CUBRID and CUBRID_DATABASES environment variables should be configured before executing testing, please refer to http://www.cubrid.org/ for configurations
+* CUBRID QA executes testing for SQL and Medium on linux is based on ha mode, so you must make sure ports in cubrid.conf, cubrid_broker.conf and cubrid_ha.conf will not conflict with another instances exist. Otherwise, start server or broker will be fail

```

Comment:
Please check the expression (on ha mode, so change ports?)

CUBRID QA executes testing for SQL and Medium on linux is based on ha mode, so you must make sure ports in cubrid.conf, cubrid_broker.conf and cubrid_ha.conf will not conflict with another instances exist. Otherwise, start server or broker will be fail